api/src/main/java/org/openmrs/Person.java

```diff
@@ -1055,4 +1055,48 @@ public void setId(Integer id) {
 		
 	}
 	
+	// Deep copy helpers
+	
+	/**
+	 * Creates deep copy of set of addresses.
+	 * 
+	 * @param personAddressesSrc Set of addresses to be copied.
+	 * @return Deep copy of given set of addresses.
+	 */
+	private void deepCopyAddresses(Set<PersonAddress> personAddressesSrc) {
+		for(PersonAddress personAddressSrc : personAddressesSrc) {
```

Comment:
No. Its the formatter's job. Does this formatter make any difference? https://wiki.openmrs.org/display/docs/Coding+Conventions